configure.ac

```diff
@@ -389,6 +389,17 @@ if test "$use_dlsym_wrapper" = "yes"; then
   AC_DEFINE([ENABLE_DLSYM_WRAPPER],[1],[Enable dlsym wrapper in libdmtcp.so])
 fi
 
+AC_ARG_ENABLE([mtcp_fixed_textaddr],
+            [AS_HELP_STRING([--enable-mtcp-fixed-textaddr],
+                            [Loads mtcp_restart at a fixed text addr, as
+                            required by MANA.])],
```

Comment:
I also like your description in the PR of the reason for this:
> In some cases, the heap assigned by the kernel is at a lower address than this mtcp_restart fixed address, causing sbrk calls to fail when they attempt to shrink heap.

Could we change the above lines to:
```
            [AS_HELP_STRING([--enable-mtcp-fixed-textaddr],
                            [Loads mtcp_restart at a fixed text addr, as
                            required by MANA.  WARNING:   In some cases,
                            the heap assigned by the kernel is at a lower
                            address than this mtcp_restart fixed address,
                           causing sbrk calls to fail when they attempt to
                           shrink heap.])],
```

Reply:
@gc00 I can add this to MANA when configuring DMTCP. But MANA no longer needs mtcp_restart for restart. I think this flag will make no difference in MANA.